meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -355,6 +355,27 @@ def get_extra_parts_label(d, id = None):
 
     return ""
 
+def get_extra_parts_mount_location(d, id = None):
```

Comment:
Should `id` really be a default argument? To me it seems mandatory, because if it's not specified, the function will always return `""`, so there is no point in calling it. Accepting that `id is None` is fine, but only when passed as an explicit argument.

Reply:
I followed the convention used in many of the related helper functions; I didn't want to rock the boat. How should I proceed here?

```
$ ag 'id = None' mender-helpers.bbclass
260:def mender_get_extra_parts_offset_by_id(d, id = None):
334:def get_extra_parts_by_id(d, id = None):
349:def get_extra_parts_label(d, id = None):
358:def get_extra_parts_mount_location(d, id = None):
391:def get_extra_parts_size_mb_by_id(d, id = None):
418:def get_extra_parts_fstab_opts(d, id = None):
```

Reply:
Right, I didn't realize it was pre-existing. Ok, let's keep it as is then.